tardis/transport/montecarlo/r_packet.py

```diff
@@ -26,6 +26,7 @@ class PacketStatus(IntEnum):
     EMITTED = 1
     REABSORBED = 2
     ADIABATIC_COOLING = 4
+    OUTSIDE_EJECTA = -1
```

Comment:
that is not true as it could also go inside. remove this and just add next_shell_id one more than there are shells or if it exits to the inside -1.

Reply:
Removed that ENUM.